core/services/ocr2/plugins/ocr2keeper/evm21/upkeepstate/store.go

```diff
@@ -45,34 +55,49 @@ type upkeepStateRecord struct {
 // In addition, performed events are fetched by the scanner on demand.
 // TODO: Add DB persistence
```

Comment:
Is this implemented now?

Reply:
That is what this PR is intended to do. I'll remove the todo